### PR TITLE
Merge master into develop

### PR DESCRIPTION
**Pull request by wileyj**
Merge master into develop using intermediary branch to resolve conflicts

### CODE REVIEW

**Review by kantai — APPROVED** [review]

File: stacks-common/src/util/log.rs [hunk 1]
```diff
@@ -238,7 +238,8 @@ fn make_logger() -> Logger {
         let plain = slog_term::PlainSyncDecorator::new(slog_term::TestStdoutWriter);
         let isatty = isatty(Stream::Stdout);
         let drain = TermFormat::new(plain, false, debug, isatty);
-        Logger::root(drain.ignore_res(), o!())
+        let logger = Logger::root(drain.ignore_res(), o!());
+        logger
```
Comment (obycode) [inline stacks-common/src/util/log.rs:242]: I think for this one, you can take the other version. It probably avoids a clippy warning.

**Review by obycode — COMMENTED** [review]
I think there's just one place I would choose the other option.

**Review by jcnelson — APPROVED** [review]

File: stacks-common/src/util/log.rs (hunk 1 above)
Comment (wileyj) [reply stacks-common/src/util/log.rs]: reverted

Comment (wileyj) [reply stacks-common/src/util/log.rs:242]: on second thought, we may want to keep this and adjust clippy - i believe this is part of a hotfix: https://github.com/stacks-network/stacks-core/commit/34703d7b63135987c296014bff7377b4115ff57f

Comment (obycode) [reply stacks-common/src/util/log.rs:242]: The change from `fuse` to `ignore_res` is the important part for the hotfix. The simplification from defining a variable and then returning it is what clippy complains about.

**Pull request merged**